WordleGame/WordleGame/SettingsViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SettingsViewController.swift
+//  WordleGame
+//
+//  Created by Alex on 15.05.22.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    @IBOutlet weak var soundSwitch: UISwitch!
+    @IBOutlet weak var vibrationSwitch: UISwitch!
+    
+    @IBOutlet weak var languagePicker: UIPickerView!
+    
+    let data = ["US", "RUS"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        languagePicker.dataSource = self
+        languagePicker.delegate = self
+        
+        soundSwitch.isOn = UserDefaults.standard.bool(forKey: "Sound Setting")
+        vibrationSwitch.isOn = UserDefaults.standard.bool(forKey: "Vibration Setting")
+        
+        soundSwitch.addTarget(self, action: #selector(soundSwitchChanged), for: UIControl.Event.valueChanged)
+        vibrationSwitch.addTarget(self, action: #selector(vibrationSwitchChanged), for: UIControl.Event.valueChanged)
+    }
+    
+    @objc func soundSwitchChanged(soundSwitch: UISwitch) {
+        if soundSwitch.isOn {
+            UserDefaults.standard.set(soundSwitch.isOn, forKey: "Sound Setting")
+        } else {
+            UserDefaults.standard.set(soundSwitch.isOn, forKey: "Sound Setting")
```

Comment:
It's the same code. No need for `if else`

Reply:
Sorry.) Resolved.